packages/fa-icon-chooser/package.json

```diff
@@ -46,7 +46,7 @@
   ],
   "scripts": {
     "build": "stencil build",
-    "start": "stencil build --dev --watch --serve",
+    "start": "stencil build --config stencil.config.dev.ts --watch --serve",
```

Comment:
This will explicitly use a separate config file (stencil.config.dev.ts) for development.
The development-specific settings (like copying the dev directory) are isolated in their own file and the base config stays clean and focused on production settings (not dev).